pyomo/contrib/pyros/uncertainty_sets.py

```diff
@@ -56,14 +67,14 @@ def __init__(self, **kwargs):
         """
         return
 
-    @property
```

Comment:
I think you still need the @property decorator (because you are overriding this with a property), but I am not surewhich order they need to be in.